x-pack/plugin/watcher/qa/rest/build.gradle

```diff
@@ -48,11 +48,13 @@ tasks.named("transformV7RestTests").configure({ task ->
   task.replaceKeyInDo("watcher.stop", "xpack-watcher.stop")
 
   task.addAllowedWarningRegex(".*_xpack/watcher/.* is deprecated.*")
+  task.addAllowedWarningRegex("\\[types removal\\].*")
+  task.replaceValueTextByKeyValue("path", "/my_index/my_type/{{ctx.watch_id}}", "/my_index/_doc/{{ctx.watch_id}}")
```

Comment:
how about we add this with a testname or a comment indicateing which test is the transformation affecting?

Reply:
done on [`68b6e3e` (#75193)](https://github.com/elastic/elasticsearch/pull/75193/commits/68b6e3e3b9aad74c2cab3976aeea582b3971a2f2)